Spanner/tests/Unit/Session/CacheSessionPoolTest.php

```diff
@@ -38,19 +38,21 @@ class CacheSessionPoolTest extends TestCase
 {
     use GrpcTestTrait;
 
-    const CACHE_KEY_TEMPLATE = 'cache-session-pool.%s.%s.%s';
+    const CACHE_KEY_TEMPLATE = CacheSessionPool::CACHE_KEY_TEMPLATE;
     const PROJECT_ID = 'project';
     const DATABASE_NAME = 'database';
     const INSTANCE_NAME = 'instance';
 
     private $time;
+    private $cacheKey;
 
     public function setUp()
     {
         $this->checkAndSkipGrpcTests();
         putenv('GOOGLE_CLOUD_SYSV_ID=U');
-        $this->time = time();
+        $this->time = 1000000000;//time();
```

Comment:
Is this just for debugging?

Reply:
Yes, this way it's easier to spot errors. Forgot to remove.